PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+Please use the following structure when proposing changes to our shared Vim configuration and make sure to complete the checklist at the end.
+
+# What
+
+Summarize your changes.
```

Comment:
`changes` => `change`. PRs are usually more effective if they are hyper-focused on one thing and do not contain multiple changes.

Reply:
Will fix.